test_build.py

```diff
@@ -70,3 +71,40 @@
 
 from skimage.io import imread
 print("skimage ok")
+
+import os
+import threading
+from http.server import BaseHTTPRequestHandler, HTTPServer
+from google.cloud import bigquery
+
+HOSTNAME = "127.0.0.1"
+PORT = 8000
+URL = "http://%s:%s" % (HOSTNAME, PORT)
+
+header_found = False
+
+class HTTPHandler(BaseHTTPRequestHandler):
+    def do_HEAD(s):
+        s.send_response(200)
+
+    def do_GET(s):
+        global header_found
+        header_found = any(k for k in s.headers if k == "X-KAGGLE-PROXY-DATA" and s.headers[k] == "test-key")
+        s.send_response(200)
+
+httpd = HTTPServer((HOSTNAME, PORT), HTTPHandler)
+thread = threading.Thread(target=httpd.serve_forever).start()
+
+client = bigquery.Client()
+
+try:
+    for ds in client.list_datasets(): pass
+except:
+    pass
+
+if header_found:
+    print("bigquery proxy ok")
+else:
+    print("bigquery proxy failed")
```

Comment:
This won't make the `docker run` of `test_build.py` (which gates the `docker tag` and `docker push`) fail.  Is there anything else in the test that does that I am missing?  If not, make sure it does that.  Perhaps simply:

```
assert header_found, "X-KAGGLE-PROXY-DATA header was missing from the BQ request".
print("bigquery kaggle header ok")
```